src/components/Container.tsx

```diff
@@ -33,6 +33,22 @@ export default function () {
         }
     }, [isPaused])
 
+    useEffect(() => {
+        document.addEventListener("keydown", handleKeyDown);
```

Comment:
Let's make these events conditional and switchable via a prop called `keyboardNavigation` (boolean) for the root import as not everyone might want their keys to respond to these changes.
You can add this prop to `GlobalCtx` type and include it inside the context object in the root `index.tsx`. Then you can import it from the `useContext` hook. For `Container.tsx` it is already imported, you would have to mention it in the de-constructor. For `SeeMore.tsx`, you'd have to import it and use as in Container.tsx. Attach the event listeners only if the prop is `true`. Let me know if you need any help here.

Reply:
I also added new things.

FYI: I haven't made any changes to the README.md file, but If you want me to update it, I can update it. This is my first contribution to a project so this reason I didn't want to make changes without asking. Thank you for your understanding.